cmd/kfulint/testdata/range-val-copy/positive_tests.go

```diff
@@ -0,0 +1,67 @@
+package checker_test
```

Comment:
Looks like merge issue.

Reply:
Why do you think so?

Reply:
I'm not sure what caused that.
We can merge this PR and then send another one that fixes it by removing/renaming incorrect things.